controller/bindings/zerotier/zerotier.ml

```diff
@@ -2,38 +2,25 @@ open Lwt
 
 let log_src = Logs.Src.create "zerotier"
 
-let base_url =
-  Uri.make
-    ~scheme:"http"
-    ~host:"localhost"
-    ~port:9993
-    ()
+let base_url = Uri.make ~scheme:"http" ~host:"localhost" ~port:9993 ()
 
 let get_authtoken () =
-  Util.read_from_file
-    log_src
-    "/var/lib/zerotier-one/authtoken.secret"
+  Util.read_from_file log_src "/var/lib/zerotier-one/authtoken.secret"
 
-type status = {
-  address: string
-}
+type status = { address : string }
 
 let get_status () =
-  Lwt_result.catch
-    (fun () ->
+  Lwt_result.catch (fun () ->
       let%lwt authtoken = get_authtoken () in
       match%lwt
         Curl.request
-          ~headers:[("X-ZT1-Auth", authtoken)]
+          ~headers:[ ("X-ZT1-Auth", authtoken) ]
           (Uri.with_path base_url "status")
       with
       | RequestSuccess (_, body) ->
           let open Ezjsonm in
-          from_string body
-          |> get_dict
-          |> List.assoc "address"
-          |> get_string
-          |> fun address -> return {address}
+          from_string body |> get_dict |> List.assoc "address" |> get_string
+          |> fun address -> return { address }
```

Comment:
This seems quite unfortunate to me, pipelines are sometimes on multiple lines, sometimes not, and overall harder for me to read.

Have you considered [`break-infix = wrap-or-vertical`](https://ocaml.org/p/ocamlformat/latest/doc/manpage_ocamlformat.html#:~:text=value%20is%20wrap.-,%2D%2Dbreak%2Dinfix%3D,-%7Bwrap%7Cfit%2Dor)?